examples/movie_view_ratings/run_on_beam.py

```diff
@@ -32,6 +32,8 @@
 FLAGS = flags.FLAGS
 flags.DEFINE_string('input_file', None, 'The file with the movie view data')
 flags.DEFINE_string('output_file', None, 'Output file')
+flags.DEFINE_integer('pre_threshold', None,
+                     'Pre threshold which is used in the DP aggregation')
```

Comment:
nit: it might be worth mentioning to which aggregations pre-threshold is currently applied.